app/models/sign_up_topic.rb

```diff
@@ -78,26 +80,13 @@ def self.reassign_topic(session_user_id, assignment_id, topic_id)
       # users_team will contain the team id of the team to which the user belongs
       users_team = SignedUpTeam.find_team_users(assignment_id, session_user_id)
       signup_record = SignedUpTeam.where(topic_id: topic_id, team_id:  users_team[0].t_id).first
-      assignment = Assignment.find(assignment_id)
+
       # if a confirmed slot is deleted then push the first waiting list member to confirmed slot if someone is on the waitlist
-      unless assignment.is_intelligent?
```

Comment:
If any code is being removed, please add a code comment of what was removed, along with your team id (E2281 I suppose)